src/libraries/System.Private.Runtime.InteropServices.JavaScript/gen/MarshalerGenerator/JavaScript.MarshalerGenerator.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please add a Directory.Build.props file to the "gen" directory that makes sure that the `IsSourceGenerator` property is set. Like https://github.com/dotnet/runtime/blob/20e8f905d05f2d8f390d0e2ef10f1a4a1289967c/src/libraries/System.Runtime.InteropServices/gen/Directory.Build.props#L11 but you only need that one property in it.

That makes sure that this project is treated correctly. cc @ericstj